parachain/pallets/inbound-queue/src/test.rs

```diff
@@ -2,325 +2,36 @@
 // SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
 use super::*;
 
-use frame_support::{
-	assert_noop, assert_ok, parameter_types,
-	traits::{ConstU128, Everything},
-	weights::IdentityFee,
-};
+use frame_support::{assert_noop, assert_ok};
 use hex_literal::hex;
-use snowbridge_beacon_primitives::{Fork, ForkVersions};
-use snowbridge_core::inbound::{Log, Proof, VerificationError};
-use snowbridge_router_primitives::inbound::MessageToXcm;
-use sp_core::{H160, H256};
+use snowbridge_core::inbound::Proof;
 use sp_keyring::AccountKeyring as Keyring;
-use sp_runtime::{
-	traits::{BlakeTwo256, IdentifyAccount, IdentityLookup, Verify},
-	BuildStorage, DispatchError, MultiSignature, TokenError,
-};
+use sp_runtime::{DispatchError, TokenError};
 use sp_std::convert::From;
-use xcm::v3::{prelude::*, MultiAssets, SendXcm};
-
-use crate::{self as inbound_queue, Error, Event as InboundQueueEvent};
-
-type Block = frame_system::mocking::MockBlock<Test>;
-
-frame_support::construct_runtime!(
-	pub enum Test
-	{
-		System: frame_system::{Pallet, Call, Storage, Event<T>},
-		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>},
-		EthereumBeaconClient: snowbridge_ethereum_beacon_client::{Pallet, Call, Storage, Event<T>},
```

Comment:
Also, I'm not sure how the beacon client pallet ended up in the mock runtime, sometime in the past. We don't really need as it, since there is a `MockVerifier`.

And the `MockVerifier` will verifyy any dummy proof, which is good enough to let us test the inbound queue.

Reply:
It is because I needed to implement the trait `BenchmarkHelper` https://github.com/Snowfork/snowbridge/blob/main/parachain/pallets/inbound-queue/src/test.rs#L140 that I added so that the inbound queue benchmarks would include the log verification in the inbound queue pallet.